src/components/dialogs/parameters/sensi/sensi-hvdcs.js

```diff
@@ -0,0 +1,169 @@
+/**
```

Comment:
This is a new file. It has to be on Typescript

Reply:
done